### ISSUE TITLE
Cannot install Nginx,Mysql, and PHP upon vagrant up

### ISSUE THREAD

**Issue by ken-master** [puphpet/puphpet#2263]
[config.zip](https://github.com/puphpet/puphpet/files/208505/config.zip)

Please see config.yaml above in zip.

upon first try on vagrant up. it did not install nginx,mysql, php, node

probabably most of the packages i selected.

OS: Windows 10
VM: Virtualbox Version 5.0.16 r105871
Vagrant: 1.8.1
Browser: Chrome Version 49.0.2623.110 m

Additional Info: upon first vagrant up

**$ vagrant up**
Bringing machine 'puphpetubuntu1404' up with 'virtualbox' provider...
==> puphpetubuntu1404: Importing base box 'puphpet/ubuntu1404-x64'...
==> puphpetubuntu1404: Matching MAC address for NAT networking...
==> puphpetubuntu1404: Checking if box 'puphpet/ubuntu1404-x64' is up to date...
==> puphpetubuntu1404: Setting the name of the VM: puphpet-trusty64_puphpetubuntu1404_1460033311530_73236
==> puphpetubuntu1404: Clearing any previously set network interfaces...
==> puphpetubuntu1404: Preparing network interfaces based on configuration...
    puphpetubuntu1404: Adapter 1: nat
    puphpetubuntu1404: Adapter 2: hostonly
==> puphpetubuntu1404: Forwarding ports...
    puphpetubuntu1404: 22 (guest) => 5795 (host) (adapter 1)
    puphpetubuntu1404: 22 (guest) => 2222 (host) (adapter 1)
==> puphpetubuntu1404: Running 'pre-boot' VM customizations...
==> puphpetubuntu1404: Booting VM...
==> puphpetubuntu1404: Waiting for machine to boot. This may take a few minutes...
    puphpetubuntu1404: SSH address: 127.0.0.1:2222
    puphpetubuntu1404: SSH username: vagrant
    puphpetubuntu1404: SSH auth method: private key
    puphpetubuntu1404: Warning: Remote connection disconnect. Retrying...
==> puphpetubuntu1404: Machine booted and ready!
==> puphpetubuntu1404: Checking for guest additions in VM...
==> puphpetubuntu1404: Setting hostname...
==> puphpetubuntu1404: Configuring and enabling network interfaces...
==> puphpetubuntu1404: Mounting shared folders...
    puphpetubuntu1404: /vagrant => C:/Users/Ken/vagrant-servers/puphpet-trusty64
    puphpetubuntu1404: /var/www/gop => C:/Users/Ken/Code/gop
    puphpetubuntu1404: /var/www/localhost => C:/Users/Ken/Code/localhost
    puphpetubuntu1404: /var/www/gop-laravel => C:/Users/Ken/Code/gop-laravel
    puphpetubuntu1404: /tmp/vagrant-puppet/modules-255525ca01232cf387052657e61666f7 => C:/Users/Ken/vagrant-se                        rvers/puphpet-trusty64/puphpet/puppet/modules
    puphpetubuntu1404: /tmp/vagrant-puppet/manifests-75f35e3bc7e32744860c4bb229c88812 => C:/Users/Ken/vagrant-                        servers/puphpet-trusty64/puphpet/puppet/manifests
==> puphpetubuntu1404: Running provisioner: shell...
    puphpetubuntu1404: Running: inline script
==> puphpetubuntu1404: Running provisioner: shell...
    puphpetubuntu1404: Running: C:/Users/Ken/AppData/Local/Temp/vagrant-shell20160407-3732-9dqhxu.sh
==> puphpetubuntu1404:
==> puphpetubuntu1404:  ____        ____  _   _ ____      _      generated using
==> puphpetubuntu1404: |  _ \ _   _|  _ \| | | |  _ \ ___| |_   ___ ___  _ __ ___
==> puphpetubuntu1404: | |_) | | | | |_) | |_| | |_) / _ \ __| / __/ _ \| '_ ` _ \
==> puphpetubuntu1404: |  __/| |_| |  __/|  _  |  __/  __/ |_ | (_| (_) | | | | | |
==> puphpetubuntu1404: |_|    \__,_|_|   |_| |_|_|   \___|\__(_)___\___/|_| |_| |_|
==> puphpetubuntu1404: Ign http://us.archive.ubuntu.com trusty InRelease
==> puphpetubuntu1404: Get:1 http://us.archive.ubuntu.com trusty-updates InRelease [65.9 kB]
==> puphpetubuntu1404: Get:2 http://security.ubuntu.com trusty-security InRelease [65.9 kB]
==> puphpetubuntu1404: Ign http://apt.puppetlabs.com trusty InRelease
==> puphpetubuntu1404: Get:3 http://security.ubuntu.com trusty-security/main Sources [110 kB]
==> puphpetubuntu1404: Get:4 http://apt.puppetlabs.com trusty Release.gpg [836 B]
==> puphpetubuntu1404: Get:5 http://us.archive.ubuntu.com trusty-backports InRelease [65.9 kB]
==> puphpetubuntu1404: Get:6 http://security.ubuntu.com trusty-security/restricted Sources [4,035 B]
==> puphpetubuntu1404: Get:7 http://security.ubuntu.com trusty-security/universe Sources [35.2 kB]
==> puphpetubuntu1404: Get:8 http://security.ubuntu.com trusty-security/multiverse Sources [2,750 B]
==> puphpetubuntu1404: Get:9 http://apt.puppetlabs.com trusty Release [54.1 kB]
==> puphpetubuntu1404: Get:10 http://security.ubuntu.com trusty-security/main amd64 Packages [455 kB]
==> puphpetubuntu1404: Get:11 http://security.ubuntu.com trusty-security/restricted amd64 Packages [13.0 kB]
==> puphpetubuntu1404: Get:12 http://security.ubuntu.com trusty-security/universe amd64 Packages [126 kB]
==> puphpetubuntu1404: Get:13 http://security.ubuntu.com trusty-security/multiverse amd64 Packages [4,991 B]
==> puphpetubuntu1404: Get:14 http://apt.puppetlabs.com trusty/PC1 Sources [7,329 B]
==> puphpetubuntu1404: Get:15 http://security.ubuntu.com trusty-security/main i386 Packages [429 kB]
==> puphpetubuntu1404: Get:16 http://apt.puppetlabs.com trusty/PC1 amd64 Packages [8,386 B]
==> puphpetubuntu1404: Get:17 http://security.ubuntu.com trusty-security/restricted i386 Packages [12.7 kB]
==> puphpetubuntu1404: Get:18 http://security.ubuntu.com trusty-security/universe i386 Packages [126 kB]
==> puphpetubuntu1404: Get:19 http://apt.puppetlabs.com trusty/PC1 i386 Packages [8,391 B]
==> puphpetubuntu1404: Get:20 http://security.ubuntu.com trusty-security/multiverse i386 Packages [5,175 B]
==> puphpetubuntu1404: Get:21 http://security.ubuntu.com trusty-security/main Translation-en [250 kB]
==> puphpetubuntu1404: Get:22 http://us.archive.ubuntu.com trusty Release.gpg [72 B]
==> puphpetubuntu1404: Get:23 http://us.archive.ubuntu.com trusty-updates/main Sources [271 kB]
==> puphpetubuntu1404: Ign http://apt.puppetlabs.com trusty/PC1 Translation-en_US
==> puphpetubuntu1404: Ign http://apt.puppetlabs.com trusty/PC1 Translation-en
==> puphpetubuntu1404: Get:24 http://security.ubuntu.com trusty-security/multiverse Translation-en [2,570 B]
==> puphpetubuntu1404: Get:25 http://security.ubuntu.com trusty-security/restricted Translation-en [3,206 B]
==> puphpetubuntu1404: Get:26 http://security.ubuntu.com trusty-security/universe Translation-en [74.5 kB]
==> puphpetubuntu1404: Get:27 http://us.archive.ubuntu.com trusty-updates/restricted Sources [5,352 B]
==> puphpetubuntu1404: Get:28 http://us.archive.ubuntu.com trusty-updates/universe Sources [152 kB]
==> puphpetubuntu1404: Get:29 http://us.archive.ubuntu.com trusty-updates/multiverse Sources [5,946 B]
==> puphpetubuntu1404: Get:30 http://us.archive.ubuntu.com trusty-updates/main amd64 Packages [754 kB]
==> puphpetubuntu1404: Get:31 http://us.archive.ubuntu.com trusty-updates/restricted amd64 Packages [15.9 kB]
==> puphpetubuntu1404: Get:32 http://us.archive.ubuntu.com trusty-updates/universe amd64 Packages [357 kB]
==> puphpetubuntu1404: Get:33 http://us.archive.ubuntu.com trusty-updates/multiverse amd64 Packages [13.2 kB]
==> puphpetubuntu1404: Get:34 http://us.archive.ubuntu.com trusty-updates/main i386 Packages [720 kB]
==> puphpetubuntu1404: Get:35 http://us.archive.ubuntu.com trusty-updates/restricted i386 Packages [15.6 kB]
==> puphpetubuntu1404: Get:36 http://us.archive.ubuntu.com trusty-updates/universe i386 Packages [359 kB]
==> puphpetubuntu1404: Get:37 http://us.archive.ubuntu.com trusty-updates/multiverse i386 Packages [13.6 kB]
==> puphpetubuntu1404: Get:38 http://us.archive.ubuntu.com trusty-updates/main Translation-en [378 kB]
==> puphpetubuntu1404: Get:39 http://us.archive.ubuntu.com trusty-updates/multiverse Translation-en [7,227 B]
==> puphpetubuntu1404: Get:40 http://us.archive.ubuntu.com trusty-updates/restricted Translation-en [3,699 B]
==> puphpetubuntu1404: Get:41 http://us.archive.ubuntu.com trusty-updates/universe Translation-en [187 kB]
==> puphpetubuntu1404: Get:42 http://us.archive.ubuntu.com trusty-backports/main Sources [8,696 B]
==> puphpetubuntu1404: Get:43 http://us.archive.ubuntu.com trusty-backports/restricted Sources [28 B]
==> puphpetubuntu1404: Get:44 http://us.archive.ubuntu.com trusty-backports/universe Sources [34.4 kB]
==> puphpetubuntu1404: Get:45 http://us.archive.ubuntu.com trusty-backports/multiverse Sources [1,898 B]
==> puphpetubuntu1404: Get:46 http://us.archive.ubuntu.com trusty-backports/main amd64 Packages [9,782 B]
==> puphpetubuntu1404: Get:47 http://us.archive.ubuntu.com trusty-backports/restricted amd64 Packages [28 B]
==> puphpetubuntu1404: Get:48 http://us.archive.ubuntu.com trusty-backports/universe amd64 Packages [41.4 kB]
==> puphpetubuntu1404: Get:49 http://us.archive.ubuntu.com trusty-backports/multiverse amd64 Packages [1,571 B                        ]
==> puphpetubuntu1404: Get:50 http://us.archive.ubuntu.com trusty-backports/main i386 Packages [9,797 B]
==> puphpetubuntu1404: Get:51 http://us.archive.ubuntu.com trusty-backports/restricted i386 Packages [28 B]
==> puphpetubuntu1404: Get:52 http://us.archive.ubuntu.com trusty-backports/universe i386 Packages [41.4 kB]
==> puphpetubuntu1404: Get:53 http://us.archive.ubuntu.com trusty-backports/multiverse i386 Packages [1,552 B]
==> puphpetubuntu1404: Get:54 http://us.archive.ubuntu.com trusty-backports/main Translation-en [5,843 B]
==> puphpetubuntu1404: Get:55 http://us.archive.ubuntu.com trusty-backports/multiverse Translation-en [1,215 B                        ]
==> puphpetubuntu1404: Get:56 http://us.archive.ubuntu.com trusty-backports/restricted Translation-en [28 B]
==> puphpetubuntu1404: Get:57 http://us.archive.ubuntu.com trusty-backports/universe Translation-en [35.9 kB]
==> puphpetubuntu1404: Get:58 http://us.archive.ubuntu.com trusty Release [11.9 kB]
==> puphpetubuntu1404: Err http://us.archive.ubuntu.com trusty Release
==> puphpetubuntu1404:
==> puphpetubuntu1404: Fetched 5,397 kB in 4min 49s (18.7 kB/s)
==> puphpetubuntu1404: Reading package lists...
==> puphpetubuntu1404: W
==> puphpetubuntu1404: :
==> puphpetubuntu1404: An error occurred during the signature verification. The repository is not updated and                         the previous index files will be used. GPG error: http://us.archive.ubuntu.com trusty Release: The following s                        ignatures couldn't be verified because the public key is not available: NO_PUBKEY 16126D3A3E5C1192
==> puphpetubuntu1404: W
==> puphpetubuntu1404: :
==> puphpetubuntu1404: Failed to fetch http://us.archive.ubuntu.com/ubuntu/dists/trusty-updates/main/binary-am                        d64/Packages  Hash Sum mismatch
==> puphpetubuntu1404: W
==> puphpetubuntu1404: :
==> puphpetubuntu1404: Failed to fetch http://us.archive.ubuntu.com/ubuntu/dists/trusty/Release
==> puphpetubuntu1404: W
==> puphpetubuntu1404: :
==> puphpetubuntu1404: Some index files failed to download. They have been ignored, or old ones used instead.
==> puphpetubuntu1404: Reading package lists...
==> puphpetubuntu1404: Building dependency tree...
==> puphpetubuntu1404: The following extra packages will be installed:
==> puphpetubuntu1404:   python-pycurl python3-pycurl python3-software-properties unattended-upgrades
==> puphpetubuntu1404: Suggested packages:
==> puphpetubuntu1404:   libcurl4-gnutls-dev python-pycurl-dbg python3-pycurl-dbg bsd-mailx
==> puphpetubuntu1404:   mail-transport-agent
==> puphpetubuntu1404: The following NEW packages will be installed:
==> puphpetubuntu1404:   python-pycurl python-software-properties python3-pycurl
==> puphpetubuntu1404:   python3-software-properties software-properties-common unattended-upgrades
==> puphpetubuntu1404: 0 upgraded, 6 newly installed, 0 to remove and 50 not upgraded.
==> puphpetubuntu1404: Need to get 169 kB of archives.
==> puphpetubuntu1404: After this operation, 1,158 kB of additional disk space will be used.
==> puphpetubuntu1404: Get:1 http://us.archive.ubuntu.com/ubuntu/ trusty/main python-pycurl amd64 7.19.3-0ubun                        tu3 [47.9 kB]
==> puphpetubuntu1404: Get:2 http://us.archive.ubuntu.com/ubuntu/ trusty-updates/main unattended-upgrades all                         0.82.1ubuntu2.3 [25.5 kB]
==> puphpetubuntu1404: Get:3 http://us.archive.ubuntu.com/ubuntu/ trusty-updates/universe python-software-prop                        erties all 0.92.37.7 [19.6 kB]
==> puphpetubuntu1404: Get:4 http://us.archive.ubuntu.com/ubuntu/ trusty/main python3-pycurl amd64 7.19.3-0ubu                        ntu3 [47.5 kB]
==> puphpetubuntu1404: Err http://us.archive.ubuntu.com/ubuntu/ trusty-updates/main python3-software-propertie                        s all 0.92.37.6
==> puphpetubuntu1404:   404  Not Found [IP: 91.189.91.14 80]
==> puphpetubuntu1404: Err http://us.archive.ubuntu.com/ubuntu/ trusty-updates/main software-properties-common                         all 0.92.37.6
==> puphpetubuntu1404:   404  Not Found [IP: 91.189.91.14 80]
==> puphpetubuntu1404: Fetched 141 kB in 10s (13.5 kB/s)
==> puphpetubuntu1404: E
==> puphpetubuntu1404: :
==> puphpetubuntu1404: Failed to fetch http://us.archive.ubuntu.com/ubuntu/pool/main/s/software-properties/pyt                        hon3-software-properties_0.92.37.6_all.deb  404  Not Found [IP: 91.189.91.14 80]
==> puphpetubuntu1404: E
==> puphpetubuntu1404: :
==> puphpetubuntu1404: Failed to fetch http://us.archive.ubuntu.com/ubuntu/pool/main/s/software-properties/sof                        tware-properties-common_0.92.37.6_all.deb  404  Not Found [IP: 91.189.91.14 80]
==> puphpetubuntu1404: E
==> puphpetubuntu1404: :
==> puphpetubuntu1404: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
==> puphpetubuntu1404: Running provisioner: shell...
    puphpetubuntu1404: Running: C:/Users/Ken/AppData/Local/Temp/vagrant-shell20160407-3732-16xcw5t.sh
==> puphpetubuntu1404: Generating public/private rsa key pair.
==> puphpetubuntu1404: Your identification has been saved in /vagrant/puphpet/files/dot/ssh/root_id_rsa.
==> puphpetubuntu1404: Your public key has been saved in /vagrant/puphpet/files/dot/ssh/root_id_rsa.pub.
==> puphpetubuntu1404: The key fingerprint is:
==> puphpetubuntu1404: f6:36:c3:7c:4a:01:0c:fd:28:d3:50:68:dd:96:ff:19 root@app
==> puphpetubuntu1404: The key's randomart image is:
==> puphpetubuntu1404: +--[ RSA 2048]----+
==> puphpetubuntu1404: |      .=.. .     |
==> puphpetubuntu1404: |      +oo +      |
==> puphpetubuntu1404: |     . oo+ .     |
==> puphpetubuntu1404: |      o o.. . E  |
==> puphpetubuntu1404: |       oS .  . o |
==> puphpetubuntu1404: |       . + .  o  |
==> puphpetubuntu1404: |          O .    |
==> puphpetubuntu1404: |         o =     |
==> puphpetubuntu1404: |          .      |
==> puphpetubuntu1404: +-----------------+
==> puphpetubuntu1404: dpkg-preconfigure: unable to re-open stdin: No such file or directory
==> puphpetubuntu1404: Your private key for SSH-based authentication has been saved to 'puphpet/files/dot/ssh/                        root_id_rsa'!
==> puphpetubuntu1404: Generating public/private rsa key pair.
==> puphpetubuntu1404: Your identification has been saved in /vagrant/puphpet/files/dot/ssh/id_rsa.
==> puphpetubuntu1404: Your public key has been saved in /vagrant/puphpet/files/dot/ssh/id_rsa.pub.
==> puphpetubuntu1404: The key fingerprint is:
==> puphpetubuntu1404: 61:d3:04:6f:6a:76:be:e5:a1:f6:d4:1a:24:06:46:45 root@app
==> puphpetubuntu1404: The key's randomart image is:
==> puphpetubuntu1404: +--[ RSA 2048]----+
==> puphpetubuntu1404: |        o+E      |
==> puphpetubuntu1404: |       . +       |
==> puphpetubuntu1404: |        * +      |
==> puphpetubuntu1404: |       o *       |
==> puphpetubuntu1404: |        S + .    |
==> puphpetubuntu1404: |       o + o .   |
==> puphpetubuntu1404: |          . = .  |
==> puphpetubuntu1404: |          .* +   |
==> puphpetubuntu1404: |         .o.+    |
==> puphpetubuntu1404: +-----------------+
==> puphpetubuntu1404: Your private key for SSH-based authentication has been saved to 'puphpet/files/dot/ssh/                        id_rsa'!
==> puphpetubuntu1404: Adding generated root key to /root/.ssh/id_rsa
==> puphpetubuntu1404: Adding generated root key to /root/.ssh/id_rsa.pub
==> puphpetubuntu1404: Adding generated root key to /root/.ssh/authorized_keys
==> puphpetubuntu1404: Adding generated key to /home/vagrant/.ssh/id_rsa
==> puphpetubuntu1404: Adding generated key to /home/vagrant/.ssh/id_rsa.pub
==> puphpetubuntu1404: Adding generated key to /home/vagrant/.ssh/authorized_keys
==> puphpetubuntu1404: Running provisioner: shell...
    puphpetubuntu1404: Running: C:/Users/Ken/AppData/Local/Temp/vagrant-shell20160407-3732-19cvvhw.sh
==> puphpetubuntu1404: Running provisioner: shell...
    puphpetubuntu1404: Running: C:/Users/Ken/AppData/Local/Temp/vagrant-shell20160407-3732-68zzv7.sh
==> puphpetubuntu1404: Successfully installed deep_merge-1.0.1
==> puphpetubuntu1404: Successfully installed activesupport-4.2.6
==> puphpetubuntu1404: Successfully installed vine-0.2
==> puphpetubuntu1404: 3 gems installed
==> puphpetubuntu1404: Running provisioner: shell...
    puphpetubuntu1404: Running: C:/Users/Ken/AppData/Local/Temp/vagrant-shell20160407-3732-6dbalo.sh
==> puphpetubuntu1404: Running files in files/exec-preprovision
==> puphpetubuntu1404: Created file /.puphpet-stuff/exec-preprovision-ran
==> puphpetubuntu1404: Finished running files in files/exec-preprovision
==> puphpetubuntu1404: To run again, delete hashes you want rerun in /.puphpet-stuff/exec-preprovision-ran or                         the whole file to rerun all
==> puphpetubuntu1404: Running provisioner: puppet...
==> puphpetubuntu1404: Running Puppet with ...
==> puphpetubuntu1404: Info: Loading facts
==> puphpetubuntu1404: Info: Loading facts
==> puphpetubuntu1404: Info: Loading facts
==> puphpetubuntu1404: Info: Loading facts
==> puphpetubuntu1404: Info: Loading facts
==> puphpetubuntu1404: Info: Loading facts
==> puphpetubuntu1404: Info: Loading facts
==> puphpetubuntu1404: Info: Loading facts
==> puphpetubuntu1404: Info: Loading facts
==> puphpetubuntu1404: Info: Loading facts
==> puphpetubuntu1404: Info: Loading facts
==> puphpetubuntu1404: Info: Loading facts
==> puphpetubuntu1404: Info: Loading facts
==> puphpetubuntu1404: Info: Loading facts
==> puphpetubuntu1404: Error: Evaluation Error: Error while evaluating a Function Call, Duplicate declaration:                         Mysql_user[root@localhost] is already declared in file /tmp/vagrant-puppet/modules-255525ca01232cf387052657e6                        1666f7/mysql/manifests/server/root_password.pp:23; cannot redeclare at /tmp/vagrant-puppet/manifests-75f35e3bc                        7e32744860c4bb229c88812/Mysql.pp:104:5 on node app.puphpet
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.

**Comment by jtreminio** [comment]
@ken-master please post your config.yaml, not the zip file.

Without it, I'm going to assume you added a `root` user declaration. Remove it, root's created by default.

**Comment by ken-master** [comment]
@jtreminio 
sorry about that. github dont allow the dot yaml file. anyways here it is.


vagrantfile:
    target: local
    vm:
        provider:
            local:
                box: puphpet/ubuntu1404-x64
                box_url: puphpet/ubuntu1404-x64
                box_version: '0'
                chosen_virtualizer: virtualbox
                virtualizers:
                    virtualbox:
                        modifyvm:
                            natdnshostresolver1: false
                        showgui: 0
                    vmware:
                        numvcpus: 1
                    parallels:
                        use_linked_clone: 0
                        check_guest_tools: 0
                        update_guest_tools: 0
                machines:
                    vflm_7h48d5ob6hmd:
                        id: puphpetubuntu1404
                        network:
                            private_network: 192.168.10.10
                            forwarded_port:
                                vflmnfp_mwll1qjzmnq6:
                                    host: '5795'
                                    guest: '22'
                        hostname: app.puphpet
                        memory: '512'
                        cpus: '1'
        provision:
            puppet:
                manifests_path: puphpet/puppet/manifests
                module_path: puphpet/puppet/modules
                options:
                    - '--verbose'
                    - '--hiera_config /vagrant/puphpet/puppet/hiera.yaml'
        synced_folder:
            vflsf_3vey4qxdhmp2:
                source: 'c:/Users/Ken/Code/gop-laravel'
                target: /var/www/gop-laravel
                sync_type: default
                smb:
                    smb_host: ''
                    smb_username: ''
                    smb_password: ''
                    mount_options:
                        dir_mode: '0775'
                        file_mode: '0664'
                rsync:
                    args:
                        - '--verbose'
                        - '--archive'
                        - '-z'
                    exclude:
                        - .vagrant/
                        - .git/
                    auto: 'true'
                owner: www-data
                group: www-data
            vflsf_7l6ubdalqlow:
                source: 'c:/Users/Ken/Code/gop'
                target: /var/www/gop
                sync_type: default
                smb:
                    smb_host: ''
                    smb_username: ''
                    smb_password: ''
                    mount_options:
                        dir_mode: '0775'
                        file_mode: '0664'
                rsync:
                    args:
                        - '--verbose'
                        - '--archive'
                        - '-z'
                    exclude:
                        - .vagrant/
                        - .git/
                    auto: 'true'
                owner: www-data
                group: www-data
            vflsf_arer6ooamyfc:
                source: 'c:/Users/Ken/Code/localhost'
                target: /var/www/localhost
                sync_type: default
                smb:
                    smb_host: ''
                    smb_username: ''
                    smb_password: ''
                    mount_options:
                        dir_mode: '0775'
                        file_mode: '0664'
                rsync:
                    args:
                        - '--verbose'
                        - '--archive'
                        - '-z'
                    exclude:
                        - .vagrant/
                        - .git/
                    auto: 'true'
                owner: www-data
                group: www-data
        usable_port_range:
            start: 10200
            stop: 10500
        post_up_message: ''
    ssh:
        host: 'false'
        port: 'false'
        private_key_path: 'false'
        username: vagrant
        guest_port: 'false'
        keep_alive: '1'
        forward_agent: 'false'
        forward_x11: 'false'
        shell: 'bash -l'
        insert_key: 'false'
    vagrant:
        host: detect
    proxy:
        http: ''
        https: ''
        ftp: ''
        no_proxy: ''
        enabled: ''
server:
    install: '1'
    packages: {  }
users_groups:
    install: '1'
    groups:
        - developers
    users: {  }
locale:
    install: '1'
    settings:
        default_locale: en_US.UTF-8
        locales:
            - en_GB.UTF-8
            - en_US.UTF-8
        timezone: Asia/Manila
firewall:
    install: '1'
    rules: {  }
cron:
    install: '1'
    jobs: {  }
nginx:
    install: '1'
    settings:
        default_vhost: 1
        proxy_buffer_size: 128k
        proxy_buffers: '4 256k'
        proxy_connect_timeout: 600s
        proxy_send_timeout: 600s
        proxy_read_timeout: 600s
        names_hash_bucket_size: 128
    upstreams: {  }
    vhosts:
        nxv_4h2hz7fqtvcd:
            server_name: gop-laravel.app
            www_root: /var/www/gop-laravel
            listen_port: '80'
            index_files:
                - index.html
                - index.htm
                - index.php
            client_max_body_size: 1m
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_port: '443'
            ssl_protocols: ''
            ssl_ciphers: ''
            rewrite_to_https: '1'
            spdy: '1'
            locations:
                nxvl_qyuu1izr5q2f:
                    location: /
                    autoindex: 'off'
                    internal: 'false'
                    try_files:
                        - $uri
                        - $uri/
                        - /index.php$is_args$args
                    fastcgi: ''
                    fastcgi_index: ''
                    fastcgi_split_path: ''
                    proxy: ''
                    proxy_redirect: ''
                nxvl_qdwr4b1yyt5e:
                    location: '~ \.php$'
                    autoindex: 'off'
                    internal: 'false'
                    try_files:
                        - $uri
                        - $uri/
                        - /index.php$is_args$args
                    fastcgi: '127.0.0.1:9000'
                    fastcgi_index: index.php
                    fastcgi_split_path: '^(.+\.php)(/.*)$'
                    fast_cgi_params_extra:
                        - 'SCRIPT_FILENAME $request_filename'
                        - 'APP_ENV dev'
                    set:
                        - '$path_info $fastcgi_path_info'
                    proxy: ''
                    proxy_redirect: ''
        nxv_187jb1b1antd:
            server_name: gop.app
            www_root: /var/www/gop
            listen_port: '80'
            index_files:
                - index.php
                - index.html
                - index.htm
            client_max_body_size: 1m
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_port: '443'
            ssl_protocols: ''
            ssl_ciphers: ''
            rewrite_to_https: '1'
            spdy: '1'
    proxies: {  }
apache:
    install: 0
    settings:
        version: 2.4
        user: www-data
        group: www-data
        default_vhost: true
        manage_user: false
        manage_group: false
        sendfile: 0
    modules:
        - proxy_fcgi
        - rewrite
    vhosts:
        av_efe5ao1z316t:
            servername: awesome.dev
            serveraliases:
                - www.awesome.dev
            docroot: /var/www/awesome
            port: '80'
            setenv:
                - 'APP_ENV dev'
            setenvif:
                - 'Authorization "(.*)" HTTP_AUTHORIZATION=$1'
            custom_fragment: ''
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_chain: ''
            ssl_certs_dir: ''
            ssl_protocol: ''
            ssl_cipher: ''
            directories:
                avd_gbdrof568hh7:
                    path: /var/www/awesome
                    options:
                        - Indexes
                        - FollowSymlinks
                        - MultiViews
                    allow_override:
                        - All
                    require:
                        - 'all granted'
                    custom_fragment: ''
                    files_match:
                        avdfm_i4nqvfmmougb:
                            path: \.php$
                            sethandler: 'proxy:fcgi://127.0.0.1:9000'
                            custom_fragment: ''
                            provider: filesmatch
                    provider: directory
php:
    install: '1'
    settings:
        version: '56'
    modules:
        php:
            - cli
            - intl
        pear: {  }
        pecl: {  }
    ini:
        display_errors: 'On'
        error_reporting: '-1'
        session.save_path: /var/lib/php/session
        date.timezone: UTC
    fpm_ini:
        error_log: /var/log/php-fpm.log
    fpm_pools:
        phpfp_e9fbj4yowooh:
            ini:
                prefix: www
                listen: '127.0.0.1:9000'
                security.limit_extensions: .php
                user: www-user
                group: www-data
    composer: '1'
    composer_home: ''
xdebug:
    install: '1'
    settings:
        xdebug.default_enable: '1'
        xdebug.remote_autostart: '0'
        xdebug.remote_connect_back: '1'
        xdebug.remote_enable: '1'
        xdebug.remote_handler: dbgp
        xdebug.remote_port: '9000'
blackfire:
    install: '0'
    settings:
        server_id: ''
        server_token: ''
        agent:
            http_proxy: ''
            https_proxy: ''
            log_file: stderr
            log_level: '1'
        php:
            agent_timeout: '0.25'
            log_file: ''
            log_level: '1'
xhprof:
    install: '0'
wpcli:
    install: '0'
    version: v0.21.1
drush:
    install: '0'
    version: 6.3.0
ruby:
    install: '1'
    versions:
        rv_s21le4mgufrc:
            default: '1'
            bundler: '1'
            version: 2.0.0-p481
            gems:
                - deep_merge
                - activesupport
                - vine
python:
    install: '1'
    packages: {  }
    versions:
        pyenv_6xuy5eo72312:
            version: 2.6.9
nodejs:
    install: '1'
    settings:
        version: '5'
    npm_packages: {  }
hhvm:
    install: '0'
    nightly: 0
    composer: '1'
    composer_home: ''
    settings: {  }
    server_ini:
        hhvm.server.host: 127.0.0.1
        hhvm.server.port: '9000'
        hhvm.log.use_log_file: '1'
        hhvm.log.file: /var/log/hhvm/error.log
    php_ini:
        display_errors: 'On'
        error_reporting: '-1'
        date.timezone: UTC
mariadb:
    install: '0'
    settings:
        version: '10.1'
        root_password: '123'
        override_options: {  }
    adminer: '1'
    users:
        mariadbnu_0u22yvnjh00y:
            name: root
            password: '123'
    databases:
        mariadbnd_smyl9zkrdaab:
            name: dbname
            sql: ''
    grants:
        mariadbng_wxarrnr5mtkl:
            user: dbuser
            table: '*.*'
            privileges:
                - ALL
mysql:
    install: '1'
    settings:
        version: '5.6'
        root_password: '123'
        override_options: {  }
    adminer: '1'
    users:
        mysqlnu_u432gozk7qaj:
            name: root
            password: root
    databases:
        mysqlnd_646xoinyxw3u:
            name: dbname
            sql: ''
    grants:
        mysqlng_vv5ix0eemaug:
            user: dbuser
            table: '*.*'
            privileges:
                - ALL
postgresql:
    install: '0'
    settings:
        global:
            encoding: UTF8
            version: '9.4'
        server:
            postgres_password: '123'
    databases: {  }
    users: {  }
    grants: {  }
    adminer: 0
mongodb:
    install: '1'
    settings:
        bind_ip: 127.0.0.1
        port: '27017'
    globals:
        version: 2.6.0
    databases: {  }
redis:
    install: '1'
    settings:
        port: '6379'
sqlite:
    install: '1'
    adminer: '1'
    databases: {  }
mailhog:
    install: '0'
    settings:
        smtp_ip: 0.0.0.0
        smtp_port: 1025
        http_ip: 0.0.0.0
        http_port: '8025'
        path: /usr/local/bin/mailhog
beanstalkd:
    install: '1'
    settings:
        listenaddress: 0.0.0.0
        listenport: '11300'
        maxjobsize: '65535'
        maxconnections: '1024'
        binlogdir: /var/lib/beanstalkd/binlog
        binlogfsync: null
        binlogsize: '10485760'
    beanstalk_console: '1'
rabbitmq:
    install: '0'
    settings:
        port: '5672'
    users: {  }
    vhosts: {  }
    plugins: {  }
elastic_search:
    install: '1'
    settings:
        version: 2.1.0
        java_install: true
    instances:
        esi_4o0i3yp2tuq0:
            name: es-01
solr:
    install: '1'
    settings:
        version: 4.10.2
        port: '8984'

**Comment by jtreminio** [comment]
```==> puphpetubuntu1404: Get:58 http://us.archive.ubuntu.com trusty Release [11.9 kB]
==> puphpetubuntu1404: Err http://us.archive.ubuntu.com trusty Release
==> puphpetubuntu1404:
==> puphpetubuntu1404: Fetched 5,397 kB in 4min 49s (18.7 kB/s)
==> puphpetubuntu1404: Reading package lists...
==> puphpetubuntu1404: W
==> puphpetubuntu1404: :
==> puphpetubuntu1404: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: http://us.archive.ubuntu.com trusty Release: The following s ignatures couldn't be verified because the public key is not available: NO_PUBKEY 16126D3A3E5C1192```

It looks like you're having issues connecting to the Ubuntu repositories. Check your firewall or network policies. Not much I can do on my end, unfortunately.

Also, for next time, please remember to paste using code formatting.

**Comment by ken-master** [comment]
i see. i actually disabled the firewall when i install my OS/. i will try it again. thanks
